INFO     root:conftest.py:125 Starting test: test_digest_auth_login_scenarios[wrong-admin].
INFO     WDM:logger.py:11 xdist worker is: gw2
INFO     WDM:logger.py:11 ====== WebDriver manager ======
INFO     WDM:logger.py:11 Get LATEST chromedriver version for google-chrome
INFO     WDM:logger.py:11 Get LATEST chromedriver version for google-chrome
INFO     WDM:logger.py:11 Driver [/home/runner/.wdm/gw2/drivers/chromedriver/linux64/142.0.7444.59/chromedriver-linux64/chromedriver] found in cache
INFO     root:conftest.py:148 Started recording: tests_recordings/gw2/test_digest_auth_login_scenarios[wrong-admin]_20251029_125402/test_digest_auth_login_scenarios[wrong-admin]_20251029_125402.mp4
INFO     root:base_page.py:100 Navigating to URL: https://the-internet.herokuapp.com/.
INFO     root:base_page.py:102 Navigation completed.
INFO     root:base_page.py:82 Waiting for page to load with indicator '('css selector', 'h1.heading')' for 10s.
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
INFO     root:base_page.py:87 Page loaded successfully with indicator '('css selector', 'h1.heading')'.
INFO     root:page_manager.py:61 Navigating to Digest Authentication page
INFO     root:base_page.py:100 Navigating to URL: https://wrong:admin@the-internet.herokuapp.com/digest_auth.
INFO     root:base_page.py:102 Navigation completed.
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
ERROR    root:base_page.py:32 Timeout after 10s for locator ('css selector', 'div#content p'): Message: 
Stacktrace:
#0 0x55d18605eaea <unknown>
#1 0x55d185aaacdb <unknown>
#2 0x55d185afd6c4 <unknown>
#3 0x55d185afd901 <unknown>
#4 0x55d185b4c8b4 <unknown>
#5 0x55d185b49c87 <unknown>
#6 0x55d185aefaca <unknown>
#7 0x55d185af07d1 <unknown>
#8 0x55d186025ab9 <unknown>
#9 0x55d186028a8c <unknown>
#10 0x55d18600ed49 <unknown>
#11 0x55d186029685 <unknown>
#12 0x55d185ff66c3 <unknown>
#13 0x55d18604b7d8 <unknown>
#14 0x55d18604b9b3 <unknown>
#15 0x55d18605da83 <unknown>
#16 0x7f676aa9caa4 <unknown>
#17 0x7f676ab29c6c <unknown>

INFO     root:test_digest_auth_login_scenarios.py:26 Page source: <html><head></head><body></body></html>
INFO     root:conftest.py:152 Stopping video recording...
INFO     utils.video_recorder:video_recorder.py:95 Video created: tests_recordings/gw2/test_digest_auth_login_scenarios[wrong-admin]_20251029_125402/test_digest_auth_login_scenarios[wrong-admin]_20251029_125402.mp4
INFO     root:conftest.py:103 Quitting driver for browser: chrome.